go.mod

```diff
@@ -4,39 +4,47 @@ go 1.13
 
 require (
 	github.com/apparentlymart/go-cidr v1.0.1
+	github.com/cloudflare/cfssl v0.0.0-20180726162950-56268a613adf // indirect
 	github.com/coreos/etcd v3.3.18+incompatible
+	github.com/coreos/rkt v1.30.0 // indirect
+	github.com/godbus/dbus v4.1.0+incompatible // indirect
+	github.com/google/certificate-transparency-go v1.0.21 // indirect
+	github.com/heketi/rest v0.0.0-20180404230133-aa6a65207413 // indirect
+	github.com/heketi/utils v0.0.0-20170317161834-435bc5bdfa64 // indirect
 	github.com/joho/godotenv v1.3.0
 	github.com/kelseyhightower/envconfig v0.0.0-20180517194557-dd1402a4d99d
 	github.com/onsi/ginkgo v1.10.1
 	github.com/onsi/gomega v1.7.0
 	github.com/patrickmn/go-cache v0.0.0-20180815053127-5633e0862627
-	github.com/projectcalico/felix v0.0.0-20200530082554-d7efcf1ff34b
-	github.com/projectcalico/libcalico-go v1.7.2-0.20200529191439-77f762f742df
-	github.com/prometheus/client_golang v0.9.4 // indirect
+	github.com/projectcalico/felix v0.0.0-20200603172703-9520b3c30731
+	github.com/projectcalico/libcalico-go v1.7.2-0.20200603171336-1b8bc835691e
 	github.com/satori/go.uuid v1.2.0
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/pflag v1.0.5
+	github.com/xlab/handysort v0.0.0-20150421192137-fb3537ed64a1 // indirect
 	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
 	golang.org/x/mod v0.3.0 // indirect
 	golang.org/x/tools v0.0.0-20200601175630-2caf76543d99 // indirect
 	gopkg.in/yaml.v2 v2.3.0 // indirect
 	honnef.co/go/tools v0.0.1-2020.1.4 // indirect
 
 	// k8s.io/api v1.16.3 is at 16d7abae0d2a
-	k8s.io/api v0.0.0
+	k8s.io/api v0.17.2
 
 	// k8s.io/apimachinery 1.16.3 is at 72ed19daf4bb
-	k8s.io/apimachinery v0.0.0
+	k8s.io/apimachinery v0.17.2
 
 	// k8s.io/apiserver 1.16.3 is at 9ca1dc586682
 	k8s.io/apiserver v0.0.0
 
 	// k8s.io/client-go 1.16.3 is at 6c5935290e33
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog v1.0.0
+	vbom.ml/util v0.0.0-20160121211510-db5cfe13f5cc // indirect
 )
 
 replace (
+	github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.4
```

Comment:
I think we have a need to update to more recent client_golang, but we'll need to address a breaking api change (I think they changed default behavior to not populate quadriles (I'm looking for the bug now).

Reply:
Ah OK, I guess the other way is to update the k8s libraries (probably the right thing to do anyway). I'm going to decline this PR and put up another one later.